### DISCUSSION TITLE
Support for Mounting Secrets Using Secrets Store CSI Driver via Helm Chart

### DISCUSSION THREAD

**Discussion by kamigerami** [community/community#160509] [Actions]
I'm working on setting up the Actions Runner Controller to utilize the Secrets Store CSI Driver for secure secret management. My goal is to fetch secrets using the githubConfigSecret value from external secret stores, which is essential for configuring the runners securely.

However, I've encountered a limitation with the current Helm chart for the Actions Runner Controller. It appears that the chart does not support specifying custom service accounts and label annotations, which are necessary for the Secrets Store CSI Driver to function properly. These configurations are crucial because they allow the CSI driver to associate the correct identity and permissions with the Kubernetes pods that the Helm chart deploys.

**Comment by oliverpark999** [comment]
Hi,
I'm having the same problem.
Did you solve it? Or did you use another solution?
